.github/workflows/test-and-build.yml

```diff
@@ -30,3 +30,9 @@ jobs:
 
     - name: Test backend
       run: yarn test
+
+    - name: Test running scheduler
+      env:
+        LIGHTDASH_CONFIG_FILE: "/home/runner/work/lightdash/lightdash/lightdash.yml"
+        LIGHTDASH_SECRET: "test"
+      run:  yarn workspace backend build && yarn workspace backend scheduler
```

Comment:
Not sure what this is going to test since we don't run anything in CI

Reply:
I dunno... maybe that the file exists and compiles ? we can remove it

Reply:
I'm inclined to remove it since `yarn workspace backend build` already builds that file.